versions.gradle

```diff
@@ -1,6 +1,8 @@
 ext {
 	jabberPluginVersion = "1.38"
 	isSnapshot = true
-	instantMessagingPluginVersion = "1.35"
+    instantMessagingPluginVersion = "1.37"
```

Comment:
Could you use tabs in those lines so that indentation is in sync with the existing ones?